src/main/java/byu/FileReader.java

```diff
@@ -0,0 +1,73 @@
+package byu;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.Scanner;
+
+import task.*;
```

Comment:
IDE might automatically do a wildcard import if you import everything from that package. Not sure if this will be allowed in the given coding standard.